android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -11,11 +12,35 @@
 public class RNCWebView extends FrameLayout {
   private static final String TAG = "RNCWebView";
   public static final int INVALID_VIEW_ID = -1;
+  public int temporaryParentNodeTag = 0;
```

Comment:
Is this the native equivalent of our "placeholder view"? This is great for our purposes. I'm curious how hard it would be to lobby for this as a generic feature in the public-facing react-native-webview

Reply:
yeah, but its just a RN `View`, not a `WebView`, since it's just used as a parent.